WordPress/Classes/Services/MediaImportService.swift

```diff
@@ -30,27 +30,23 @@ open class MediaExportService: LocalCoreDataService {
 
     // MARK: - Instance methods
 
-    /// Creates a Media object with an absoluteLocalURL for a PHAsset's data, asynchronously.
-    /// - paramater blog: a blog object to where the media object will be added to.
-    /// - paramater post: an optional post object to where the media object will be attached to.
-    /// - parameter onCompletion: Called if the Media was successfully created and the asset's data exported to an absoluteLocalURL.
+    /// Imports media from a PHAsset to the Media object, asynchronously.
```

Comment:
Can you do a find and replace in this file for `paramater` -> `parameter`? There look to be quite a few of them in here.

Reply:
Will do!! Good catch